lib/omniauth/strategies/google_oauth2.rb

```diff
@@ -135,6 +135,10 @@ def image_url
           u.path = u.path.gsub('//', '/')
         end
 
+        # strip query parameters (like the default ?z=50) 
```

Comment:
Just to eliminate confusion. You wrote in your comment `?z=50` instead of `?sz=50`.